src/testUtils/java/org/broadinstitute/hellbender/testutils/BaseTest.java

```diff
@@ -318,45 +318,6 @@ public static void log(final String message) {
 
     private static final double DEFAULT_FLOAT_TOLERANCE = 1e-1;
 
-
-    /**
-     * Checks whether two double array contain the same values or not.
-     * @param actual actual produced array.
-     * @param expected expected array.
-     * @param tolerance maximum difference between double value to be consider equivalent.
-     */
-    protected static void assertEqualsDoubleArray(final double[] actual, final double[] expected, final double tolerance) {
-        if (expected == null)
-            Assert.assertNull(actual);
-        else {
-            Assert.assertNotNull(actual);
-            Assert.assertEquals(actual.length,expected.length,"array length");
-        }
-        for (int i = 0; i < actual.length; i++)
-            Assert.assertEquals(actual[i],expected[i],tolerance,"array position " + i);
-    }
-
-    /**
-     * Checks whether two long arrays contain the same values or not.
-     * @param actual actual produced array.
-     * @param expected expected array.
-     */
-    protected static void assertEqualsLongArray(final long[] actual, final long[] expected) {
```

Comment:
I think this might be redundant with testngs Assert.assertEqual now.  I *think* it does reflection and checks if the objects are arrays and then uses Arrays.equals which is an element by element comparison.

Reply:
Done (and yes, it does reflection)